prow/scripts/cluster-integration/kyma-gardener-integration.sh

```diff
@@ -108,20 +141,40 @@ kyma provision gardener \
         --target-provider azure --secret "${GARDENER_KYMA_PROW_PROVIDER_SECRET_NAME}" \
         --name "${CLUSTER_NAME}" --project "${GARDENER_KYMA_PROW_PROJECT_NAME}" --credentials "${GARDENER_KYMA_PROW_KUBECONFIG}" \
         --region "${GARDENER_REGION}" -t "${MACHINE_TYPE}" --disk-size 35 --disk-type=Standard_LRS --extra vnetcidr="10.250.0.0/16" \
-        --nodes 1 \
+        --nodes 4 \
         --kube-version=${CLUSTER_VERSION}
 )
 
 shout "Installing Kyma"
 date
 
+shout "Downloading Kyma installer CR"
+curl -L --silent --fail --show-error "https://raw.githubusercontent.com/sayanh/kyma/integration-azure-event-hubs/installation/resources/installer-cr-gardener-azure.yaml.tpl" \
```

Comment:
Change it to master once https://github.com/kyma-project/kyma/pull/7205/files gets merged

Reply:
Done